### PR TITLE
[Automation] Fix smoke test project and VFX test failure

### PR DESCRIPTION
**Pull request by sophiaaar**
When the HDRP package is in a project, it needs an HDRP asset to be in the graphics settings, otherwise a dialog box appears when trying to build. This causes automated builds to break because it requires manual input.

Also, the VFX project has a test using Timeline. I missed this when I was removing dependencies in the projects, and the test started failing.

---
### Purpose of this PR
* Adds in render pipeline assets to the smoke test project
* Sets the project to be HDRP
* Fixes the failing VFX test

---
### Testing status

**Manual Tests**: What did you do?
- [x] Opened test project + Run graphic tests locally
- [x] Built a player
- [ ] Checked new UI names with UX convention
- [ ] Tested UI multi-edition + Undo/Redo + Prefab overrides
- [ ] C# and shader warnings (supress shader cache to see them)
- [ ] Checked new resources path for the reloader (in devloper mode, you have a button at end of resources that check the pathes)
- Other: 

**Yamato**: (Select your branch):
VFX passing: https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/automation%252Ffix-smoke-test/.yamato%252Fupm-ci-vfxmain.yml%2523All_VFXMain_fast-trunk/726295/job

The Smoke Test project is now failing because of the Ads/UTR failure, which is expected. The rest of the tests pass: https://yamato.prd.cds.internal.unity3d.com/jobs/78-ScriptableRenderPipeline/tree/automation%252Ffix-smoke-test/.yamato%252Fupm-ci-abv.yml%2523all_smoke_tests/726499/job/pipeline

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: TestProjects/VisualEffectGraph/Packages/manifest.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "com.unity.ide.rider": "1.1.1",
     "com.unity.ide.visualstudio": "1.0.11",
     "com.unity.ide.vscode": "1.1.3",
+	"com.unity.timeline": "1.2.0",
```
Comment (sebastienlagarde) [inline TestProjects/VisualEffectGraph/Packages/manifest.json]: do we need to add timeline package here?

Comment (sebastienlagarde) [reply TestProjects/VisualEffectGraph/Packages/manifest.json]: isn't smoke test about not fixing package version?

Comment (sebastienlagarde) [reply TestProjects/VisualEffectGraph/Packages/manifest.json]: ah yes, it is not part of smoke test :)

Comment (sophiaaar) [reply TestProjects/VisualEffectGraph/Packages/manifest.json]: yes, thats how the vfx test is fixed :)

**Pull request merged**